src/library/masq.js

```diff
@@ -100,17 +104,47 @@ class Masq {
     const privateProfile = {
       ...publicProfile,
       firstname: profile.firstname,
-      lastname: profile.lastname,
-      hashedPassphrase: hashedPassphrase
+      lastname: profile.lastname
     }
 
     // Create a DB for this profile
     const db = openOrCreateDB(id)
     await dbReady(db)
-    await db.putAsync('/', privateProfile)
+
+    // we do not use this.encryptAndPut because this value is not encrypted
+    await db.putAsync('/profile/protectedMK', protectedMK)
+    // We do not use this.encryptAndPut method because the master key
+    // is not stored in this.profileDB (only set in openProfile)
+    await put(db, this.masterKey, '/', privateProfile)
```

Comment:
can you store at /profile to avoid the error we discussed in the sync PR please ?